### PR TITLE
feat: add docker container for vcf2cytosure

### PR DESCRIPTION
**Pull request by ashwini06**
### This PR:

Previously `vcf2cytosure` is added to the annotate container in PR #650. But there exists a problem loading `vcf2cytosure` in this container

aa1

The problem might be with the loading entry points for vcf2cytosure installation and its installation exited..

Removed: vcf2cytosure from `annotate` container

Added: New Docker container for vcf2cytosure. Related to #858 

### Review and tests: 
- [ ] Tests pass
- [ ] Code review
- [ ] New code is executed and covered by tests, and test approve

### CODE REVIEW

File: BALSAMIC/containers/vcf2cytosure/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+LABEL about.description="Bioinformatic analysis pipeline for somatic mutations in cancer"
+
+ENV PATH="/opt/vcf2cytosure/bin:${PATH}"
+
+RUN apk add --no-cache bash gcc git python3
+
+RUN cd /opt \
+    && git clone https://github.com/NBISweden/vcf2cytosure.git \
+    && cd /opt/vcf2cytosure \
+    && pip install -e .
```
Comment (ivadym) [inline BALSAMIC/containers/vcf2cytosure/Dockerfile:17]:
```suggestion
    && cd /opt/${CONTAINER_NAME} \ \
    && pip install --no-cache-dir .
```

File: BALSAMIC/containers/vcf2cytosure/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,17 @@
+FROM continuumio/miniconda3:4.10.3-alpine
+
+LABEL base.image="continuumio/miniconda3:4.10.3-alpine"
+LABEL about.home="https://github.com/Clinical-Genomics/BALSAMIC"
+LABEL about.documentation="https://balsamic.readthedocs.io/"
+LABEL about.license="MIT License (MIT)"
+LABEL about.maintainer="Ashwini Jeggari ashwini dot jeggari at scilifelab dot se"
+LABEL about.description="Bioinformatic analysis pipeline for somatic mutations in cancer"
+
+ENV PATH="/opt/vcf2cytosure/bin:${PATH}"
```
Comment (ivadym) [inline BALSAMIC/containers/vcf2cytosure/Dockerfile:10]:
```suggestion
ARG CONTAINER_NAME
ENV PATH="/opt/${CONTAINER_NAME}/bin:${PATH}"
ENV PYTHONPATH="/opt/${CONTAINER_NAME}"
```

**Review by ivadym — CHANGES_REQUESTED** [review]
👍 

Regarding your comment, it seems that Python does not know where to look for the required modules, but it should be solved setting explicitly the `PYTHONPATH` environment variable. I tested it locally with the changes that I suggested and it works:

Screen Shot 2022-03-02 at 20 26 40

**Comment by ashwini06** [comment]
> 👍
> 
> Regarding your comment, it seems that Python does not know where to look for the required modules, but it should be solved setting explicitly the `PYTHONPATH` environment variable. I tested it locally with the changes that I suggested and it works:
> 
> Screen Shot 2022-03-02 at 20 26 40

Thanks @ivadym for your suggestions :) It works fine now with your suggestions

File: BALSAMIC/containers/vcf2cytosure/Dockerfile (hunk 2 above)
Comment (ashwini06) [reply BALSAMIC/containers/vcf2cytosure/Dockerfile]: Fixed!

File: BALSAMIC/containers/vcf2cytosure/Dockerfile (hunk 1 above)
Comment (ashwini06) [reply BALSAMIC/containers/vcf2cytosure/Dockerfile]: FixeD!

**Comment by ashwini06** [comment]
@ivadym @khurrammaqbool : Ready for review :speaker:

**Review by ivadym — APPROVED** [review]
👍

**Pull request merged**